app/gui/src/project-view/composables/events.ts

```diff
@@ -140,6 +154,8 @@ export function modKey(e: KeyboardEvent | MouseEvent): boolean {
   return isMacLike ? e.metaKey : e.ctrlKey
 }
 
+export type UnrefElement = ToValue<Element | undefined | null | VueInstance>
```

Comment:
Type seems like being something returned from `unrefElement`, but it's not the case - it isn't even mentioned in the method's signature.